### PR TITLE
install-golang: go version should be 1.16

### LINKED ISSUE #19: Upgrade Go version to 1.16
In https://zz-jason.gitbook.io/tidb-dev-guide/get-started/install-golang, the version of Go is 1.13, but the master has already turned into 1.16.

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What issue does this PR solve?

- close #19 

### What is changed:

Update go version in document to 1.16.

### CODE REVIEW

File: get-started/install-golang.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ To build TiDB from source code, you need to install Go in your development envir
...
 
-## Install Go 1.13
+## Install Go 1.16
 
 Most TiDB developers are using Linux or MacOS. If you are using Windows 10, to have a better experience, it is recommended to [install Windows Subsystem for Linux 2 \(WSL2\)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) on Windows 10 first. TiDB should compile and run on Windows 10, but it is not expected to be deployed on Windows, where you might encounter many compatibility problems. In that case, you can either figure out the solutions on your own or share the problems on our [TiDB Internals forum](https://internals.tidb.io/).
 
-Currently, TiDB uses Go 1.13 to compile the code, but we plan to upgrade the compiler to Go 1.16 in the near future. To install Go 1.13, go to [Go's download page](https://golang.org/dl/), choose version 1.13, and then follow the [install instruction](https://golang.org/doc/install).
+Currently, TiDB uses Go 1.16 to compile the code, but we plan to upgrade the compiler to Go 1.16 in the near future. To install Go 1.16, go to [Go's download page](https://golang.org/dl/), choose version 1.16, and then follow the [install instruction](https://golang.org/doc/install).
```
Comment (ichn-hu) [inline get-started/install-golang.md:14]: s/but we plan to upgrade the compiler to Go 1.16 in the near future//

**Review by ichn-hu — CHANGES_REQUESTED** [review]

**Comment by ichn-hu** [comment]
Rest LGTM

File: get-started/install-golang.md (hunk 1 above)
Comment (tisonkun) [reply get-started/install-golang.md:14]:
```suggestion
Currently, TiDB uses Go 1.16 to compile the code. To install Go 1.16, go to [Go's download page](https://golang.org/dl/), choose version 1.16, and then follow the [install instruction](https://golang.org/doc/install).
```

Comment (tisonkun) [reply get-started/install-golang.md:14]: Thanks!

**Comment by tisonkun** [comment]
@ichn-hu Updated. PTAL.

**Review by ichn-hu — APPROVED** [review]

**Review by zz-jason — APPROVED** [review]
LGTM

**Comment by zz-jason** [comment]
@all-contributors please add @ichn-hu for review

**Pull request merged**